diff --git a/src/broadcom/vulkan/v3dv_device.c b/src/broadcom/vulkan/v3dv_device.c
index 547eeb1102..7bd380bb5f 100644
--- a/src/broadcom/vulkan/v3dv_device.c
+++ b/src/broadcom/vulkan/v3dv_device.c
@@ -75,13 +75,12 @@ v3dv_EnumerateInstanceVersion(uint32_t *pApiVersion)
     return VK_SUCCESS;
 }
 
-#if defined(VK_USE_PLATFORM_WIN32_KHR) ||   \
-    defined(VK_USE_PLATFORM_WAYLAND_KHR) || \
-    defined(VK_USE_PLATFORM_XCB_KHR) ||     \
-    defined(VK_USE_PLATFORM_XLIB_KHR) ||    \
-    defined(VK_USE_PLATFORM_DISPLAY_KHR)
-#define V3DV_USE_WSI_PLATFORM
-#endif
+
+#define V3DV_HAS_SURFACE (VK_USE_PLATFORM_WIN32_KHR ||   \
+                          VK_USE_PLATFORM_WAYLAND_KHR || \
+                          VK_USE_PLATFORM_XCB_KHR ||     \
+                          VK_USE_PLATFORM_XLIB_KHR ||    \
+                          VK_USE_PLATFORM_DISPLAY_KHR)
 
 static const struct vk_instance_extension_table instance_extensions = {
    .KHR_device_group_creation           = true,
@@ -93,7 +92,7 @@ static const struct vk_instance_extension_table instance_extensions = {
    .KHR_external_memory_capabilities    = true,
    .KHR_external_semaphore_capabilities = true,
    .KHR_get_physical_device_properties2 = true,
-#ifdef V3DV_USE_WSI_PLATFORM
+#ifdef V3DV_HAS_SURFACE
    .KHR_get_surface_capabilities2       = true,
    .KHR_surface                         = true,
    .KHR_surface_protected_capabilities  = true,
