# http://lists.busybox.net/pipermail/buildroot/2020-August/288635.html
diff --git a/setup.py b/setup.py
index 2b600ca..887dabd 100644
--- a/setup.py
+++ b/setup.py
@@ -43,4 +43,4 @@ setup(name             = 'RPi.GPIO',
       url              = 'http://sourceforge.net/projects/raspberry-gpio-python/',
       classifiers      = classifiers,
       packages         = ['RPi','RPi.GPIO'],
-      ext_modules      = [Extension('RPi._GPIO', ['source/py_gpio.c', 'source/c_gpio.c', 'source/cpuinfo.c', 'source/event_gpio.c', 'source/soft_pwm.c', 'source/py_pwm.c', 'source/common.c', 'source/constants.c'])])
+      ext_modules      = [Extension('RPi._GPIO', ['source/py_gpio.c', 'source/c_gpio.c', 'source/cpuinfo.c', 'source/event_gpio.c', 'source/soft_pwm.c', 'source/py_pwm.c', 'source/common.c', 'source/constants.c'],extra_link_args=['-Wl,--allow-multiple-definition'] )])
