diff -u -r src_orig/pyudev/core.py src/pyudev/core.py
--- a/src/pyudev/core.py	2020-11-29 13:03:56.676982392 +0100
+++ b/src/pyudev/core.py	2020-11-29 13:04:25.189455784 +0100
@@ -58,7 +58,7 @@
         """
         Create a new context.
         """
-        self._libudev = load_ctypes_library('udev', SIGNATURES, ERROR_CHECKERS)
+        self._libudev = load_ctypes_library('libudev.so', SIGNATURES, ERROR_CHECKERS)
         self._as_parameter_ = self._libudev.udev_new()
 
     def __del__(self):
diff -u -r src_orig/pyudev/_os/pipe.py src/pyudev/_os/pipe.py
--- a/src/pyudev/_os/pipe.py	2020-11-29 13:03:56.676982392 +0100
+++ b/src/pyudev/_os/pipe.py	2020-11-29 13:04:15.701298252 +0100
@@ -91,7 +91,7 @@
         return os.pipe2  # pylint: disable=no-member
     else:
         try:
-            libc = load_ctypes_library("libc", SIGNATURES, ERROR_CHECKERS)
+            libc = load_ctypes_library('libc.so', SIGNATURES, ERROR_CHECKERS)
             return (partial(_pipe2_ctypes, libc)
                     if hasattr(libc, 'pipe2') else _pipe2_by_pipe)
         except ImportError:
